### PR TITLE
Added spanish deck building to buildDeck()

### PR DESCRIPTION
**Pull request by IStallcup**
Added boolean to measure type of deck, some logic to build new spanish version of deck. If I remember our discussion correctly, we need to write a new card class/make card abstract (use template design pattern), any thoughts on this?

### CODE REVIEW

**Comment by IStallcup** [comment]
CI check should fail, as the commit references stuff that hasn't been written yet.

File: src/main/java/models/Game.java [hunk 1]
```diff
@@ -15,6 +15,8 @@
 
     public boolean error = false;
 
+    public boolean isAmerican = true;
```
Comment (leel8on) [inline src/main/java/models/Game.java:18]: Your patriotism is inspiring :)

File: src/main/java/models/Game.java [hunk 2]
```diff
@@ -23,11 +25,23 @@ public Game(){
...
+            for (int i = 2; i < 15; i++){
+                deck.add(new Card(i,Suit.Clubs));
+                deck.add(new Card(i,Suit.Hearts));
+                deck.add(new Card(i,Suit.Diamonds));
+                deck.add(new Card(i,Suit.Spades));
+            }
+        }
+        else if (isAmerican == false)
+        {
+            for (int i = 1; i < 11; i++){
```
Comment (leel8on) [inline src/main/java/models/Game.java:39]: The difference in the conditions for the for loop are part of handling the different number of cards, correct?

Comment (IStallcup) [inline src/main/java/models/Game.java:39]: Yes indeed they are.

File: src/main/java/models/Game.java (hunk 1 above)
Comment (IStallcup) [inline src/main/java/models/Game.java:18]: American Flag

**Comment by athai** [comment]
What do you mean by "the commit references stuff that hasn't been written yet"?

**Comment by IStallcup** [comment]
The code references yet unwritten spanish deck requirements.

**Comment by kannabanana** [comment]
I think we're going to nuke the build deck function inside the game class since the spanish and english deck need to have separate ones and use inheritance.

**Pull request merged**